File: Leetcode 2801-2900/2833/2833.py
# Leetcode 2833: Furthest Point From Origin
# https://leetcode.com/problems/furthest-point-from-origin/
# Solved on 24th of April, 2026
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:
        """
        Calculates the maximum possible distance from the origin after replacing all '_' with 'L' or 'R'.

        :param moves: A string consisting of 'L', 'R', and '_' characters.
        :return: The maximum absolute distance from the origin (0).
        """
        currentPosition = 0
        blankSpaces = 0

        for currentMove in moves:
            if currentMove == 'L':
                currentPosition -= 1
            elif currentMove == 'R':
                currentPosition += 1
            else:
                blankSpaces += 1

        return abs(currentPosition) + blankSpaces

File: Leetcode 2801-2900/2833/2833.java
// Leetcode 2833: Furthest Point From Origin
// https://leetcode.com/problems/furthest-point-from-origin/
// Solved on 24th of April, 2026
class Solution {
    /**
     * Calculates the furthest distance from the origin after a series of moves.
     * 
     * @param moves A string consisting of 'L', 'R', and '_', representing moves.
     * @return The maximum absolute distance from the origin.
     */
    public int furthestDistanceFromOrigin(String moves) {
        int distance = 0;
        int blankCount = 0;
        
        for (int i = 0; i < moves.length(); i++) {
            char move = moves.charAt(i);
            if (move == 'L') {
                distance--;
            } else if (move == 'R') {
                distance++;
            } else {
                blankCount++;
            }
        }
        
        return Math.abs(distance) + blankCount;
    }
}